src/views/blocks/attendees/view-link.php

```diff
@@ -58,7 +59,8 @@
 
 $link = $view->get_tickets_page_url( $event_id, $is_event_page );
 
-$message = esc_html( sprintf( __( 'You have %1s for this %2s.', 'event-tickets' ), implode( _x( ' and ', 'separator if there are both RSVPs and Tickets', 'event-tickets' ), $counters ), $events_label_singular ) );
+// Translators: 1: number of RSVPs and/or Tickets with accompanying ticket type text, 2: post type label
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 50cf0d1e80fdf010b903e5d9ebe77bf0</sub>